resources/public/style.css

```diff
@@ -308,4 +296,26 @@ fieldset {
 
 #plain-password {
 	width: 10em;
-}
\ No newline at end of file
+}
+
+/* SYNTAX HIGHLIGHTING */
+
+/* Theme: default, source: http://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.5.0/styles/default.min.css */
+.hljs{display:block;overflow-x:auto;padding:0.5em;background:#F0F0F0}.hljs,.hljs-subst{color:#444}.hljs-comment{color:#888888}.hljs-keyword,.hljs-attribute,.hljs-selector-tag,.hljs-meta-keyword,.hljs-doctag,.hljs-name{font-weight:bold}.hljs-type,.hljs-string,.hljs-number,.hljs-selector-id,.hljs-selector-class,.hljs-quote,.hljs-template-tag,.hljs-deletion{color:#880000}.hljs-title,.hljs-section{color:#880000;font-weight:bold}.hljs-regexp,.hljs-symbol,.hljs-variable,.hljs-template-variable,.hljs-link,.hljs-selector-attr,.hljs-selector-pseudo{color:#BC6060}.hljs-literal{color:#78A960}.hljs-built_in,.hljs-bullet,.hljs-code,.hljs-addition{color:#397300}.hljs-meta{color:#1f7199}.hljs-meta-string{color:#4d99bf}.hljs-emphasis{font-style:italic}.hljs-strong{font-weight:bold}
+
+pre, code, code.hljs {
+	font-family: monospace;
+	border-radius: 3px;
+	background: #fcfcfc;
```

Comment:
I tested the result on my local machine and it looks beautiful: http://imgur.com/jDUU2Hs.
But I noticed that `#fcfcfc`is barely visible on a MacBook display (which are usually considered calibrated well enough even for professional photography). What kind of monitor do you use? I wonder whether you see the same I do (namely almost nothing :) ). I'd consider either make it darker or maybe even drop the background color completely. In this case we could increase the left padding to make the code snippets visually stand out.

Another thing I wondered is why did you decrease the font size? I remember, the `1.05em` fix was in orderto make the code font size be approximately the same size as normal text. Now with the hack below you removed this and I can see, that the code font size is now smaller (see my screenshot).

However, all of these are really minor things and it's up to you whether you want to invest any time into it or not. It's totally ok to ignore my comment — I'll improve it in the future when I get the time.